### PR TITLE
Very important fixes. Getting ready to launch!

### PR DESCRIPTION
**Pull request by UladBohdan**
* Dante2 is updated so that "kek" issue is gone
* Improve about page vacancies section:
screen shot 2018-07-27 at 03 34 40

* Fix action buttons padding:
screen shot 2018-07-27 at 03 57 18

* Add confirm modal & remove article action:
screen shot 2018-07-29 at 01 23 54

* Add 404 page:
screen shot 2018-07-29 at 18 20 04

* Add 500 page:
screen shot 2018-07-29 at 18 35 30

> Depends on [babajka-markup#49](https://github.com/babajka/babajka-markup/pull/49) styles

### CODE REVIEW

**Comment by UladBohdan** [comment]
[images are not rendered properly in the Editor: add an image to article -> save -> back to editor (it's broken) -> back to article page (it's ok)](https://trello.com/c/o9e3rUgd/249-images-are-not-rendered-properly-in-the-editor-add-an-image-to-article-save-back-to-editor-its-broken-back-to-article-page-its-o)

**Review by Drapegnik — DISMISSED** [review]

**Comment by Drapegnik** [comment]
[save / remove / open buttons under the Editor are too close to the footer, looks awful](https://trello.com/c/v133CT4V/251-save-remove-open-buttons-under-the-editor-are-too-close-to-the-footer-looks-awful)

**Comment by Drapegnik** [comment]
[save / remove / open buttons under the Editor are too close to the footer, looks awful](https://trello.com/c/v133CT4V/251-save-remove-open-buttons-under-the-editor-are-too-close-to-the-footer-looks-awful)

**Comment by Drapegnik** [comment]
[to add 'remove article' button on Frontend](https://trello.com/c/ryufaBgF/219-to-add-remove-article-button-on-frontend)

**Comment by tyndria** [comment]
[Chrome tries to autocomplete videoUrl field with postal index, why?](https://trello.com/c/Cnjd4AyE/248-chrome-tries-to-autocomplete-videourl-field-with-postal-index-why)

**Comment by Drapegnik** [comment]
[to implement 404 page on Frontend](https://trello.com/c/J30dI2Ko/181-to-implement-404-page-on-frontend)

**Comment by Drapegnik** [comment]
[google analytics script repeating bug](https://trello.com/c/xb03e8Lq/262-google-analytics-script-repeating-bug)

File: components/articles/PublicArticle/PublicArticle.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { connect } from 'react-redux';
+import _ from 'lodash';
```
Comment (Drapegnik) [inline components/articles/PublicArticle/PublicArticle.js]: please use `import foo from 'lodash/foo'` [method](https://stackoverflow.com/questions/35250500/correct-way-to-import-lodash)

**Review by Drapegnik — CHANGES_REQUESTED** [review]

File: components/articles/PublicArticle/PublicArticle.js (hunk 1 above)
Comment (Drapegnik) [reply components/articles/PublicArticle/PublicArticle.js]: one line - `+527.73 KB`:

screen shot 2018-08-05 at 12 12 41screen shot 2018-08-05 at 12 11 29

**Review by Drapegnik — APPROVED** [review]

**Pull request merged**